app/main.py

```diff
@@ -15,5 +15,12 @@
     "Sergio": 3,
 }
 collection_of_coins = {1, 2, 25}
+sorted_variables = {"mutable" : [], "immutable" : []}
+all = [lucky_number, pi, one_is_a_prime_number, name, my_favourite_films, profile_info, marks, collection_of_coins]
+for var in all:
+    if isinstance(var, (list, dict, set)):
+        sorted_variables["mutable"].append(var)
+    else:
+        sorted_variables["immutable"].append(var)
```

Comment:
This is a good example of using isinstance() function and tuple to check for multiple types. It's a clean and efficient way to sort variables into mutable and immutable. Good job!